src/contrib/contribution-policy.rst

```diff
@@ -0,0 +1,43 @@
+Contribution policy
+===================
+
+There are many ways you can contribute to the System Security Services Daemon.
+This page should provide some basic pointers.
```

Comment:
I think this line can be removed, s/This page should provide some basic pointers//

Reply:
Fixed